src/vs/workbench/electron-browser/main.contribution.ts

```diff
@@ -104,6 +104,16 @@ let workbenchProperties: { [path: string]: IJSONSchema; } = {
 		'description': nls.localize('showEditorTabs', "Controls if opened editors should show in tabs or not."),
 		'default': true
 	},
+	'workbench.editor.tabDescription': {
```

Comment:
I wonder how clear this setting for users is. I know we call this "description" internally, but unsure if people will relate this to the small text next to the main label of a tab. 

We already have `window.title`, maybe this should be `workbench.editor.tabSubtitle` ?